src/plugin/svipc/sysvipcwrappers.cpp

```diff
@@ -108,14 +122,22 @@ void *shmat(int shmid, const void *shmaddr, int shmflg)
 }
 
 extern "C"
+bool dmtcp_svipc_inside_shmdt()
```

Comment:
The rest of this file uses `extern "C"`. Do I need to make a special case for this function?